src/data/option/n_ary.lean

```diff
@@ -170,4 +172,16 @@ lemma map_map₂_right_anticomm {f : α → β' → γ} {g : β → β'} {f' : 
   map₂ f a (b.map g) = (map₂ f' b a).map g' :=
 by cases a; cases b; simp [h_right_anticomm]
 
+/-- If `a` is a left identity for a binary operation `f`, then `some a` is a left identity for
+`option.map₂ f`. -/
+lemma map₂_left_identity {f : α → β → β} {a : α} (h : ∀ b, f a b = b) (o : option β) :
```

Comment:
Could you add the same lemmas to the other `n_ary` files? Some of them, are already ported, sorry, but the you could also golf `data.set.pointwise` and `data.finset.pointwise`.

Reply:
Done. Feel free to golf either in this PR or in another one.